[jsscripting] Globals cannot be injected automatically on Windows

The JSScripting add-on doesn't seem to load the global variables (the default behavior) for UI-created rules on Windows.
Note that this seems to happen _even_ when the _Use Built-in Global Variables_ option in Settings > Other Services > JS Scripting is set to _Do Not Use Built-in Variables_.

On the same snapshot on Linux, it seems to work.

## Expected Behavior
The global variables should be loaded with no exception.

## Current Behavior
This exception appears in the logs on the first execution of a script, regardless of the option above:
```
[ERROR] [g.internal.OpenhabGraalJSScriptEngine] - Could not inject global script
javax.script.ScriptException: org.graalvm.polyglot.PolyglotException: TypeError: Cannot load CommonJS module: '@jsscripting-globals'
        at com.oracle.truffle.js.scriptengine.GraalJSScriptEngine.toScriptException(GraalJSScriptEngine.java:483) ~[?:?]
        at com.oracle.truffle.js.scriptengine.GraalJSScriptEngine.eval(GraalJSScriptEngine.java:460) ~[?:?]
        at com.oracle.truffle.js.scriptengine.GraalJSScriptEngine.eval(GraalJSScriptEngine.java:426) ~[?:?]
        at javax.script.AbstractScriptEngine.eval(AbstractScriptEngine.java:264) ~[java.scripting:?]
<snip>
Caused by: org.graalvm.polyglot.PolyglotException: TypeError: Cannot load CommonJS module: '@jsscripting-globals'
        at com.oracle.truffle.polyglot.PolyglotMapAndFunction.apply(PolyglotMapAndFunction.java:46) ~[?:?]
        at org.openhab.automation.jsscripting.internal.OpenhabGraalJSScriptEngine.lambda$3(OpenhabGraalJSScriptEngine.java:178) ~[?:?]
        at java.util.Optional.orElseGet(Optional.java:369) ~[?:?]
        at org.openhab.automation.jsscripting.internal.OpenhabGraalJSScriptEngine.lambda$1(OpenhabGraalJSScriptEngine.java:178) ~[?:?]
        at <js>.:program(<eval>:1) ~[?:?]
        at org.graalvm.polyglot.Context.eval(Context.java:379) ~[?:?]
        at com.oracle.truffle.js.scriptengine.GraalJSScriptEngine.eval(GraalJSScriptEngine.java:458) ~[?:?]
        ... 71 more
```

## Possible Solution
A workaround is to add explitely:
```js
var runtime = require('@runtime');
```
to all scripts and use that, this works, but the exception is ugly and cannot be avoided.


## Steps to Reproduce (for Bugs)
1. Install the JSScripting add-on
2. Create a ECMAScript 2021 script in the UI (Settings > Scripts) or a rule with a script action set to ECMAScript 2021.
3. Run it

This might be because of path separators, for example here:
https://github.com/openhab/openhab-addons/blob/a6cb90f9e5fbdb22e6f1900c7316c0f106eb7811/bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/OpenhabGraalJSScriptEngine.java#L68


## Context
The JSScripting add-on which is a big feature of openHAB 3.2 doesn't work well in the simplest scenario on Windows (e.g. for testing purposes).

## Your Environment
<!-- Include as many relevant details about the environment you experienced the bug in -->
On Windows, Zulu 11, fresh openHAB 3.2.0-SNAPSHOT - Build 2632,

--- a/bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/OpenhabGraalJSScriptEngine.java
+++ b/bundles/org.openhab.automation.jsscripting/src/main/java/org/openhab/automation/jsscripting/internal/OpenhabGraalJSScriptEngine.java
@@ -14,6 +14,7 @@ package org.openhab.automation.jsscripting.internal;
 
 import static org.openhab.core.automation.module.script.ScriptEngineFactory.*;
 
+import java.io.File;
 import java.io.IOException;
 import java.io.InputStream;
 import java.nio.channels.SeekableByteChannel;
@@ -65,7 +66,7 @@ public class OpenhabGraalJSScriptEngine extends InvocationInterceptingScriptEngi
     private static final String GLOBAL_REQUIRE = "require(\"@jsscripting-globals\");";
     private static final String REQUIRE_WRAPPER_NAME = "__wraprequire__";
     // final CommonJS search path for our library
-    private static final Path LOCAL_NODE_PATH = Paths.get("/node_modules");
+    private static final Path LOCAL_NODE_PATH = Paths.get(File.separator + "node_modules");
 
     // these fields start as null because they are populated on first use
     private @NonNullByDefault({}) String engineIdentifier;

